test/Main.purs

```diff
@@ -55,3 +58,13 @@ main = do
   checkOrd prxHugeInt
   checkSemiring prxHugeInt
   checkRing prxHugeInt
+  log "Checking from/to Num...\n"
+  quickCheck toFromNumIsIdentity
+  log "Checking from/to Int...\n"
+  quickCheck toFromIntIsIdentity
+
+toFromNumIsIdentity :: Number -> Boolean
+toFromNumIsIdentity n = HN.toNumber (HN.fromNumber n) == n
+
+toFromIntIsIdentity :: Int -> Boolean
+toFromIntIsIdentity n = HI.toInt (HI.fromInt n) == Just n
```

Comment:
Thanks for adding these tests! Do these fail without the change you added?

Reply:
I think only the `Int` one fails.